src/nvapi_gpu.cpp

```diff
@@ -829,6 +835,24 @@ extern "C" {
         if (!nvapiAdapterRegistry->IsAdapter(adapter))
             return ExpectedPhysicalGpuHandle(n);
 
+        if (!adapter->HasNvml() && env::needsNvmlSuccess()) {
+            switch (pClkFreqs->version) {
+                case NV_GPU_CLOCK_FREQUENCIES_VER_1: {
+                    auto pClkFreqsV1 = reinterpret_cast<NV_GPU_CLOCK_FREQUENCIES_V1*>(pClkFreqs);
+                    *pClkFreqsV1 = {};
+                    break;
+                }
+                case NV_GPU_CLOCK_FREQUENCIES_VER_2: // Both versions use the same NV_GPU_CLOCK_FREQUENCIES_V2 struct
+                case NV_GPU_CLOCK_FREQUENCIES_VER_3:
+                    *pClkFreqs = {};
```

Comment:
Yeah, I'm not really happy with this either. May be it is worth at some point to investigate if the game is properly initializing everything, then we could just skip the memset for all three work arounds, but currently I don't own this title.

Reply:
Nah, I'm not sure if it's worth the headache, just something to keep in mind in case the game ever starts behaving strangely after some update due to this.